### ISSUE TITLE
Align labels to top margin of a textarea

### ISSUE THREAD

**Issue by pathumego** [trufont/trufont#232]
<img width="391" alt="screen shot 2016-02-05 at 8 03 41 pm" src="https://cloud.githubusercontent.com/assets/6846609/12857739/c36a6912-cc43-11e5-8d93-2c84e5ee8e9f.png">

**Comment by madig** [comment]
Hm. I'd need to fiddle with the layout box to get this looking right, `setAlignment(Qt.AlignRight | Qt.AlignTop)` doesn't cut it.

**Comment by madig** [comment]
And neither do spacers it seems. Converting it into a QFormLayout works, but messes with the box alignment versus the grid.

**Comment by madig** [comment]
Hah! I managed to get this done using cascading QFormLayouts:
![bildschirmfoto vom 2016-02-06 14-54-16](https://cloud.githubusercontent.com/assets/380829/12866991/93be2e8a-cce1-11e5-9a30-f2b11cbaa83f.png)
I'm going to try giving other dialogs the same treatment.

**Comment by pathumego** [comment]
@madig Looks perfect. Im opening a new issue to discuss Font Info window layout

**Comment by madig** [comment]
Good idea. I'm not entirely convinced by how the info dialog is organized.

**Comment by madig** [comment]
(to clarify: I'm waiting on opinions in #234 before I invest time in changing something that will be changed again)

**Comment by madig** [comment]
Will be fixed in new fontinfo dialog.
